### DISCUSSION TITLE
More visible mesh overlaps in Editor

### DISCUSSION THREAD

**Discussion by larinius** [stride3d/stride#1339] [Ideas]
It's hard to position objects in Editor, because selected mesh is bright yellow and hard to see what is actual object location.
Example on screen - 1st mesh is above ground, 2nd mesh is below ground, but it looks the same.

![image](https://user-images.githubusercontent.com/5310985/155861070-e54b1820-5424-4158-8db5-f35f25dc10dd.png)

![image](https://user-images.githubusercontent.com/5310985/155861062-b3823fed-2858-4030-b891-dc57457554e2.png)

**Comment by Eideren** [comment]
Yes, that's something that can be improved, you should open an enhancement issue instead
